lib/src/client/transport/xhr_transport.dart

```diff
@@ -96,8 +106,13 @@ class XhrTransportStream implements GrpcTransportStream {
             onError: _onError, onDone: _incomingMessages.close);
   }
 
+  bool _checkContentType(String contentType) {
+    return _validContentTypePrefix
+        .any((value) => contentType.startsWith(value));
```

Comment:
any reason not to write this as `.any(contentType.startsWith)`?

Reply:
Done.